### ISSUE TITLE
ifort compilation totnum_R error

### ISSUE THREAD

**Issue by iamc** [mesonepigreco/CellConstructor#16]
Trying to compile with the adapted setup.py file [`setup-EB-intel-Python.py.zip`](https://github.com/mesonepigreco/CellConstructor/files/4691279/setup-EB-intel-Python.py.zip)
 for Ekhi cluster intel toolchain I get the following error (see full [`install.log`](https://github.com/mesonepigreco/CellConstructor/files/4691278/install.log)), which seems to be a compilation, not setup.py related, problem:

```
analyzevars: charselector={'len': '45'} unhandled.FModules/third_order_ASR.f90(12): error #6415: This name cannot be assigned this data type because it conflicts with prior uses of the name.   [TOTNUM_R]
    integer, intent(in) :: xR_list(3,totnum_R),totnum_R ! list different R vectors
-----------------------------------------------^
compilation aborted for FModules/third_order_ASR.f90 (code 1)
FModules/third_order_ASR.f90(12): error #6415: This name cannot be assigned this data type because it conflicts with prior uses of the name.   [TOTNUM_R]
    integer, intent(in) :: xR_list(3,totnum_R),totnum_R ! list different R vectors
-----------------------------------------------^
```

In order to reproduce:

```
# ml ASE/3.19.0-intel-2018b-Python-3.6.6
# cd ~/tests/sscha
# virtualenv --system-site-packages sscha-intel-2018b-Python-3.6.6
# source sscha-intel-2018b-Python-3.6.6/bin/activate
# git clone git@github.com:mesonepigreco/CellConstructor.git
# cd CellConstructor
## copy attached setup-EB-intel-Python.py here
# python setup-EB-intel-Python.py install --record files.txt 2>&1 | tee install.log
```

**Comment by mesonepigreco** [comment]
Hi @iamc ,
Did you fixed this issue in the pull request? I think you just need to define totnum_R in the previous line. If you think we are ready, I will merge the pull request

**Comment by iamc** [comment]
Yes, issue fixed in PR #17
